gdal/src/main/scala/geotrellis/contrib/vlm/gdal/GDALWarpOptions.scala

```diff
@@ -255,6 +256,8 @@ case class GDALWarpOptions(
 object GDALWarpOptions {
   val EMPTY = GDALWarpOptions()
 
+  implicit def lift2Monad[F[_]: Monad](options: GDALWarpOptions): F[GDALWarpOptions] = Monad[F].pure(options)
```

Comment:
This implicit actually, makes the fact that options should be wrapped now invisible.